cpp/dolfinx/mesh/cell_types.cpp

```diff
@@ -280,24 +95,18 @@ mesh::CellType mesh::cell_facet_type(mesh::CellType type)
 Eigen::Array<int, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>
 mesh::get_entity_vertices(mesh::CellType type, int dim)
 {
-  switch (type)
-  {
-  // case mesh::CellType::point:
-  //   return create_entities_point(e, v);
-  case mesh::CellType::interval:
-    return get_entity_vertices_interval(dim);
-  case mesh::CellType::triangle:
-    return get_entity_vertices_triangle(dim);
-  case mesh::CellType::tetrahedron:
-    return get_entity_vertices_tetrahedron(dim);
-  case mesh::CellType::quadrilateral:
-    return get_entity_vertices_quadrilateral(dim);
-  case mesh::CellType::hexahedron:
-    return get_entity_vertices_hexahedron(dim);
-  default:
-    throw std::runtime_error("Unsupported cell type.");
-    return Eigen::Array<int, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>();
-  }
+  assert(dim >= 0);
```

Comment:
This should be runtime error, not assert, if you want to check for it. But I would just skip the check.